books/free-programming-books-te.md

```diff
@@ -15,6 +15,16 @@
 * [Introduction to C \| Telugu](https://www.computerintelugu.com/2012/11/cmenu.html) - Sivanaadh Baazi Karampudi
 
 
+### <a id="mern"></a>MERN Web development
+
+* [Web Development Full Course in Telugu (MERN)](https://www.youtube.com/playlist?list=PL2Kd-KQLppEFbfdJHywOu6b_GAjn3LxE1&si=btWBIKe1EbMntfys) - Dodagatta Nihar
```

Comment:
```suggestion
* [Web Development Full Course in Telugu (MERN)](https://www.youtube.com/playlist?list=PL2Kd-KQLppEFbfdJHywOu6b_GAjn3LxE1) - Dodagatta Nihar
```
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes